media-types.md

```diff
@@ -5,8 +5,9 @@ The following `mediaType` MIME types are used by the formats described here, and
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
 - `application/vnd.oci.image.manifest.list.v1+json`: [Manifest list](manifest.md#manifest-list)
 - `application/vnd.oci.image.manifest.v1+json`: [Image manifest format](manifest.md#image-manifest)
-- `application/vnd.oci.image.layer.tar+gzip`: ["Layer", as a gzipped tar archive](layer.md)
 - `application/vnd.oci.image.config.v1+json`: [Container config JSON](serialization.md)
+- `application/vnd.oci.image.layer.tar+gzip`: ["Layer", as a gzipped tar archive](layer.md)
+- `application/vnd.oci.image.layer.foreign.tar+gzip`: ["Layer", as a gzipped tar that MUST NOT be distributed](layer.md#foreign-layers)
```

Comment:
On Fri, Sep 09, 2016 at 01:13:31AM -0700, Jonathan Boulle wrote:
> >  - `application/vnd.oci.image.config.v1+json`: [Container config JSON](serialization.md)
> > +- `application/vnd.oci.image.layer.tar+gzip`: ["Layer", as a gzipped tar archive](layer.md)
> > +- `application/vnd.oci.image.layer.foreign.tar+gzip`: ["Layer", as a gzipped tar that MUST NOT be distributed](layer.md#foreign-layers)
>
> The MUST here seems to be contradicting line 167 of the other
> doc. Can we remove it or maybe change it to MAY?

I don't think we want MAY here.  Can we end both this and the other section with:

  … has distribution restrictions.